### ISSUE TITLE
Video Not Available

### ISSUE THREAD

**Issue by thisguyshouldworkforus** [JuanBindez/pytubefix#50]
**Describe the bug**
Attempts to download anything results in "Video Not Available" error.

### Attempting a simple command on a "Slo Mo Guys" video
```bash
/opt/projects/mytube/venv/bin/python3
>>> import pytubefix
>>> VIDEO = "https://www.youtube.com/watch?v=2wiMOUIx_5o"
>>> yt = pytubefix.YouTube(str(VIDEO), use_oauth=True, allow_oauth_cache=True)
>>> yt.title
'Video Not Available'
```

### Also, tried it without auth, thinking perhaps just my account was getting rate limited or something.
```bash
/opt/projects/mytube/venv/bin/python3
>>> import pytubefix
>>> VIDEO = "https://www.youtube.com/watch?v=2wiMOUIx_5o"
>>> yt = pytubefix.YouTube(str(VIDEO))
>>> yt.title
'Video Not Available'
```

**Expected behavior**
I would expect it to download videos as its been doing. This suddenly happened last night.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Red Hat Enterprise Linux release 9.3 (Plow)
 - Python Version 3.11.5
 - Pytubefix Version 2.0.0 (intentionally locked at that version)

**Additional context**
This is how I'm implementing your project into mine
  - https://github.com/thisguyshouldworkforus/mytube/blob/main/TEMPLATE.py
  - https://github.com/thisguyshouldworkforus/mytube/blob/main/libs/functions.py

**Comment by JuanBindez** [comment]
try version 3.0.0

**Comment by NannoSilver** [comment]
At my ending it works properly:


```
video_id = '2wiMOUIx_5o'
url = f'https://youtube.com/watch?v={video_id}'

yt = YouTube(url)
duration = yt.length
print(f'duration:      {duration} seconds')

title = yt.title
print(f'title:      {title}')
```


Output:
```
duration:      968 seconds
title:      Superman’s “Bullet vs Eyeball” Shot at 200,000 FPS -  The Slow Mo Guys
```


Tested at Windows, Python 3.9, Pytubefix 3.0.0 final release.

**Comment by thisguyshouldworkforus** [comment]
The "Video Not Available" issue is resolved in v3.0.0, but [as mentioned here (issue #51)](https://github.com/JuanBindez/pytubefix/issues/51) any version above 2.0.0 there is now an `AttributeError` on `yt.publish_date`
```python
>>> (yt.publish_date).strftime
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'strftime'
```

This bug does not exist in v2.0.0

**Comment by JuanBindez** [comment]
try:

```python
from pytubefix import YouTube

video_url = 'https://www.youtube.com/watch?v=jDwVkXVHIqg&t=25s'

yt = YouTube (video_url)

publish_date = yt.publish_date
print(publish_date)

```
